src/main/java/org/jabref/logic/exporter/SavePreferences.java

```diff
@@ -37,15 +36,14 @@ private SavePreferences(Boolean saveInOriginalOrder, SaveOrderConfig saveOrder,
         this.reformatFile = reformatFile;
         this.fieldWriterPreferences = fieldWriterPreferences;
         this.globalCiteKeyPattern = globalCiteKeyPattern;
-        this.generateBibtexKeysBeforeSaving = generateBibtexKeysBeforeSaving;
         this.bibtexKeyPatternPreferences = bibtexKeyPatternPreferences;
     }
 
     public SavePreferences(Boolean saveInOriginalOrder, SaveOrderConfig saveOrder, Charset encoding,
                            DatabaseSaveType saveType, Boolean takeMetadataSaveOrderInAccount, Boolean reformatFile,
                            FieldWriterPreferences fieldWriterPreferences, GlobalBibtexKeyPattern globalCiteKeyPattern,
-                           Boolean generateBibtexKeysBeforeSaving, BibtexKeyPatternPreferences bibtexKeyPatternPreferences) {
-        this(saveInOriginalOrder, saveOrder, encoding, true, saveType, takeMetadataSaveOrderInAccount, reformatFile, fieldWriterPreferences, globalCiteKeyPattern, generateBibtexKeysBeforeSaving, bibtexKeyPatternPreferences);
+                           BibtexKeyPatternPreferences bibtexKeyPatternPreferences) {
```

Comment:
The `globalCiteKeyPattern` is now also included in the `bibtexKeyPatternPreferences` right?

Reply:
Thanks, I totally overlooked this one.
GlobalCiteKeyPattern was in fact part of `BibtexKeyPatternPreferences` already before. I removed it now.